Sources/Models/GHOAuthPayload.swift

```diff
@@ -0,0 +1,22 @@
+import DiscordBM
+import JWTKit
+
+/// This Payload will get sent along with the OAuth redirect to the GitHub OAuth page,
+/// specifically inside the `state` query parameter.
+/// This is used to verify the user's identity when they come back from GitHub 
+/// and that the request is not forged.
+public struct GHOAuthPayload: JWTPayload {
+    public let discordID: UserSnowflake
+    public let interactionID: InteractionSnowflake
```

Comment:
This was my bad, change this to `InteractionToken: String`:
```suggestion
    public let interactionToken: String
```